libmemcached/csl/scanner.cc

```diff
@@ -463,76 +463,77 @@ static void yy_fatal_error (yyconst char msg[] ,yyscan_t yyscanner );
 	yyg->yy_c_buf_p = yy_cp;
 
 /* %% [4.0] data tables for the DFA and the user's section 1 definitions go here */
-#define YY_NUM_RULES 66
-#define YY_END_OF_BUFFER 67
+#define YY_NUM_RULES 67
+#define YY_END_OF_BUFFER 68
 /* This struct is not used in this scanner,
    but its presence is necessary. */
 struct yy_trans_info
 	{
 	flex_int32_t yy_verify;
 	flex_int32_t yy_nxt;
 	};
-static yyconst flex_int16_t yy_accept[534] =
+static yyconst flex_int16_t yy_accept[547] =
     {   0,
-        0,    0,   67,   65,    5,    5,    1,   65,   65,   65,
-        2,   65,   65,   65,   65,   65,   65,   65,   65,   65,
-       65,   65,   65,   65,   65,   65,    0,   64,    0,   49,
+        0,    0,   68,   66,    5,    5,    1,   66,   66,   66,
+        2,   66,   66,   66,   66,   66,   66,   66,   66,   66,
+       66,   66,   66,   66,   66,   66,    0,   65,    0,   50,
```

Comment:
YES! Finally someone has fixed this up

Reply:
I know - it's so nice to finally be tackling this annoying technical debt. Finally now our velocity can start increasing!